### ISSUE TITLE
stackablectl could not be installed in docs tests

### ISSUE THREAD

**Issue by backstreetkiwi** [stackabletech/ci#43]
@fhennig encountered this behaviour in the docs tests:
```
installing Operators with stackablectl
stackablectl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by stackablectl)
stackablectl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by stackablectl)
stackablectl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by stackablectl)
```

**Comment by backstreetkiwi** [comment]
The T2 testdriver (docker.stackable.tech/t2-testdriver) in which the tests run inside Jenkins is based on a python:3 image (which is in turn some debian image). The newest GLIBC available there is 2.31.x. 
As the documentation test only use stackablectl to install the operators, they should be fine with an older version, so I pinned it to stackablectl:0.6.0. Once the python:3 image supports a newer GLIBC, we will be able to unpin it.

**Comment by fhennig** [comment]
Thanks!
